frontend/src/app/pages/marker-page/marker-page.component.html

```diff
@@ -52,6 +52,32 @@ <h3>Actions</h3>
                         </button>
                     </span>
                 </div>
+                <span *ngFor="let action of selectorActions">
+                    <mat-card *ngIf="action.type === ActionType.SLIDER" class="slider">
+                        <mat-card-content>
+                            <h4>{{ action.name }}:</h4>
+
+                            <mat-slider *ngIf="action.type === ActionType.SLIDER"
+                                        (input)="action.trigger(($event)['value'])"
+                                        [thumbLabel]="true"
+                                        [max]="action.options['sliderMax']"
+                                        [min]="action.options['sliderMin']"
+                                        [value]="action.options['sliderDefault']"
+                                        [step]="action.options['sliderStep']">
+                            </mat-slider>
+                        </mat-card-content>
+                    </mat-card>
+                    <mat-card *ngIf="action.type === ActionType.COLOR_PICKER" class="color-picker">
+                        <mat-card-content>
+                            <h4>{{ action.name }}:</h4>
+
+                            <mcc-color-picker (change)="action.trigger($event)" [hideHexForms]="true" [hideUsedColors]="true"
```

Comment:
For me, User shouldn't change color of Labels on his own. Why would he/she need to do it? Let's do it for them and preset the color on Backend side for each Label Tag (defined in configuration file). It's part of Issue #357.

I don't know why your Issue was telling to do so... I'm sorry - should've check its description on my own... I'll address the problem with Issues description directly and we'll do something about it.

But don't worry - JS code will definitely be reused as we will need it for #357 💪 Do not remove it!